java-tools/OsmAndServer/src/main/java/net/osmand/server/api/services/WikiService.java

```diff
@@ -45,6 +45,10 @@ public class WikiService {
 
 	private static final int LIMIT_QUERY = 1000;
 	private static final int LIMITI_QUERY = 25;
+	
+	private static final Map<Integer, Set<String>> EXCLUDED_POI_TYPES_BY_ZOOM = Map.of(14, Set.of("shop"));
+	private static final Map<Integer, Set<String>> EXCLUDED_POI_SUBTYPES_BY_ZOOM = Map.of(14, Set.of("artwork", "commercial", "memorial", "monument"));
```

Comment:
Some artwork / memorial could be very popular https://www.wikidata.org/wiki/Q185382